modules/ROOT/pages/getting-started.adoc

```diff
@@ -114,6 +114,7 @@ systemd:
       contents: |
         [Unit]
         Description=Run single node etcd
+        After=network-online.target
```

Comment:
This needs to be coupled with a `Wants=network-online.target`, as this target is not activated by default and thus needs to be pulled-in where needed.

Reply:
Ok, I tested that works.